packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/feature_onboarding/ui_components/register_form.dart

```diff
@@ -0,0 +1,136 @@
+{{> licence.dart }}
+
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/flutter_rx_bloc.dart';
+import 'package:flutter_rx_bloc/rx_form.dart';
+import 'package:provider/provider.dart';
+import 'package:widget_toolkit/widget_toolkit.dart' hide ErrorModel;
+
+import '../../app_extensions.dart';
+import '../../base/common_ui_components/app_error_modal_widget.dart';
+import '../../base/extensions/async_snapshot_extensions.dart';
+import '../../base/extensions/error_model_field_translations.dart';
+import '../blocs/onboarding_bloc.dart';
+
+class RegisterForm extends StatefulWidget {
+  const RegisterForm({
+    this.title = 'Enter your registration credentials',
```

Comment:
All strings must be localized

Reply:
There's no reason to do this here, removed